### PR TITLE
Overview credentials

### PR DESCRIPTION
**Pull request by valentinBraeutigam**
- added overview credential definition page including backend connection
- some minor refactoring in frontend and backend
- tests for filtered-table component

Release manager: @Idontker

### CODE REVIEW

File: src/digitalIdentity-backend/src/test/java/didentity/amos/digitalIdentity/controller/AuthenticationControllerTests.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import didentity.amos.digitalIdentity.enums.UserRole;
 import didentity.amos.digitalIdentity.model.User;
 
-public class AuthentificationControllerTests {
+public class AuthenticationControllerTests {
```
Comment (Idontker) [inline src/digitalIdentity-backend/src/test/java/didentity/amos/digitalIdentity/controller/AuthenticationControllerTests.java:9]:
```suggestion
public class AuthenticationControllerTest {
```

File: src/digitalIdentity-frontend/src/app/components/filtered-table/filtered-table.component.spec.ts [hunk 2]
```diff
@@ -1,25 +1,200 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
```
Comment (Idontker) [inline src/digitalIdentity-frontend/src/app/components/filtered-table/filtered-table.component.spec.ts:1]: Did not get coauthored eventhough I wrote a lot as well :| 
Just in a diffrent place which will not be visible as its use is deprecated.

File: src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/model/User.java [hunk 3]
```diff
@@ -32,6 +32,8 @@ public class User {
     @Column(length = 2048)
     private String invitationUrl;
 
+    private String state;
```
[the remark this answers is not in the document]
Comment (valentinBraeutigam) [reply src/digitalIdentity-backend/src/main/java/didentity/amos/digitalIdentity/model/User.java:35]: Okay I will remove it

File: src/digitalIdentity-frontend/src/app/components/filtered-table/filtered-table.component.spec.ts (hunk 2 above)
Comment (valentinBraeutigam) [reply src/digitalIdentity-frontend/src/app/components/filtered-table/filtered-table.component.spec.ts:1]: I'm really sorry about that :( I wanted to, but I forgot it

**Pull request merged**